shared/api/access.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+// access represents who has access to an instance or project
+//
+// swagger:model
+//
+// API extension: instances.
+type Access []AccessEntry
+
+type AccessEntry struct {
```

Comment:
This struct also needs to have a swagger:model comment so it's properly included in the swagger model.